src/js/components/Filter.js

```diff
@@ -1,8 +1,7 @@
 // (C) Copyright 2014-2015 Hewlett Packard Enterprise Development LP
 
 import React, { Component, PropTypes } from 'react';
-import pick from 'lodash/object/pick';
-import keys from 'lodash/object/keys';
+import { FormattedMessage } from 'react-intl';
```

Comment:
Actually it is OK if you are using npm 3.x. It flattens dependency tree as much as it can. Since `react-intl` comes with Grommet, you can safely import it. However, It's going to be a problem if the packages are installed with npm 2.x, you are right.

I made the changes.